app/src/main/java/io/gnosis/safe/ui/settings/app/fiat/FiatListAdapter.kt

```diff
@@ -36,7 +36,7 @@ class FiatListAdapter : RecyclerView.Adapter<FiatListAdapter.FiatViewHolder>() {
 
     override fun onBindViewHolder(holder: FiatViewHolder, position: Int) {
         val item = items[position]
-        val displayName = Currency.getInstance(item).displayName
+        val displayName = Currency.getInstance(item).getDisplayName(Locale("en", Locale.getDefault().country))
```

Comment:
Ah cool you fixed the issue with the fiat names #1190  ?

Reply:
yes

Reply:
thanks :D